dxr/app.py

```diff
@@ -249,6 +251,23 @@ def raw(tree, path):
     return send_file(data_file, mimetype=guess_type(path)[0])
 
 
+@dxr_blueprint.route('/<tree>/rawrev/<revision>/<path:path>')
```

Comment:
I'm not in love with the user-visible name "rawrev", since nowhere else do we mash two words up against each other without any divider.